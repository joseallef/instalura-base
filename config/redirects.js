// eslint-disable-next-line no-undef
module.exports = [
  {
    source: '/login/',
    destination: '/app/login/',
    permanent: true,
  },
  ]