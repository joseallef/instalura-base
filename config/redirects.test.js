import redirects from './redirects';

console.log(redirects);

// Test Driven Development
describe('config/redirects', () => {
    test('reders all courrent redirects', () => {
      expect(redirects).toMatchSnapshot();
    })
});