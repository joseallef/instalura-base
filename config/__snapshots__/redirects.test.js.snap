// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config/redirects reders all courrent redirects 1`] = `
Array [
  Object {
    "destination": "/app/login/",
    "permanent": true,
    "source": "/login/",
  },
]
`;
